src/content/1.7/scale/optimizations.md

```diff
@@ -20,15 +20,61 @@ of concurrent users you want to simulate during the bench.
 
 Make sure Zend Opcache is enabled.
 
+> Note this configuration of PHP should be used in Production environments only.
+
 Use the following settings to optimize the performances:
 ```text
+[Date]
+date.timezone = UTC
+
+[Session]
+session.auto_start = Off
+
+[PHP]
+short_open_tag = Off
+realpath_cache_size = 4096K
+realpath_cache_ttl = 600
+apc.enable_cli = 1
+display_errors = Off
+
+magic_quotes_gpc = off
+memory_limit = 512M
```

Comment:
add a comment because you can set more memory if you have only one server with more memory than 2Gb :)